client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoConnection.java

```diff
@@ -469,7 +469,7 @@ public CallableStatement prepareCall(String sql, int resultSetType, int resultSe
     public PreparedStatement prepareStatement(String sql, int autoGeneratedKeys)
             throws SQLException
     {
-        if (autoGeneratedKeys != Statement.RETURN_GENERATED_KEYS) {
+        if (autoGeneratedKeys != Statement.NO_GENERATED_KEYS) {
```

Comment:
Please add a test to `TestJdbcConnection`.

Reply:
If we want to test, we apparently need a `PreparedStatement` with Hive. I don't see which request to submit under Hive (I don't know Hive)?

Reply:
TestJdbcConnection already uses Hive connector.

Reply:
> TestJdbcConnection already uses Hive connector.

Reply:
@ebyhr I just added the tests